corehq/apps/reports/views.py

```diff
@@ -425,7 +425,8 @@ def hq_update_saved_export(req, domain):
     index = int(req.POST['index'])
     group_config = get_document_or_404(HQGroupExportConfiguration, domain, group_id)
     config, schema = group_config.all_exports[index]
-    rebuild_export_task.delay(group_id, index)
+    filter = SerializableFunction(instances)
```

Comment:
I'm confused by this behavior, since I thought that deleted forms were not included in saved exports, which I would have assumed was also handled by `instances`. Were those also being included in saved exports? Or if not, where was that filtering happening?